tests/Valit.Tests/String/String_IsEqualTo_Tests.cs

```diff
@@ -36,7 +36,7 @@ public void String_IsEqualTo_Returns_Proper_Result_For_Left_Value(string value,
         [Theory]
         [InlineData("text", false)]
         [InlineData("other", false)]
-        [InlineData("", false)]
+        [InlineData("", true)]
```

Comment:
This seems to make more sense now: "" is now considered equal to "". Previously the test was saying comparing "" to "" should result in false.